tests/e2e/slurm/slurm_test.go

```diff
@@ -8,13 +8,11 @@ import (
 	"testing"
 )
 
-var log = logger.New("e2e-slurm")
 var fun *e2e.Funnel
 var runTest = flag.Bool("run-test", false, "run e2e tests with dockerized scheduler")
+var log = logger.NewLogger("slurm", logger.DebugConfig())
```

Comment:
This file should be consistent with gridengine, htcondor, etc.  In those files you changed `TestMain` to use `logger.Debug` rather than declaring a global logger.